Packs/FeedOffice365/README.md

```diff
@@ -0,0 +1,69 @@
+## Overview
+---
+
+Use the Office365 feed integration to get indicators from the feed.
+This integration was integrated and tested with version xx of Office365 Feed
```

Comment:
please update `xx` to an actual version

Reply:
it has no version, just a url you access, e.g:
https://endpoints.office.com/endpoints/Worldwide?ServiceAreas=SharePoint&ClientRequestId=<uuid>